src/commonmark/en/content/implementation/support.md

```diff
@@ -28,58 +27,62 @@ implementation, database development and maintenance. The change log and
-code hosting, functionality specifications, bug tracking and
-notifications. The *Bazaar* version control system is tightly integrated
-with Launchpad and is required for checking out the source code.
+The primary DHIS2 collaboration platform is the *DHIS2 Community of Practice*. The site can
+be accessed at *https://community.dhis2.org/* and is based on the Discourse platform.
 
-The various source code branches including trunk and release branches
-can be browsed at *http://code.launchpad.net/dhis2*
+The Community of Practice is used to facilitate community support for DHIS2 user issues, as
+well as to help identify potential bugs in existing software versions and feature requests
+for future versions. It is also a place where members of the community can share stories,
+best practices, and challenges from their DHIS2 implementations, collaborate with others
+on projects, and offer their services to the larger community. Users can set up their Community
+of Practice accounts based on individual preferences for notification settings, and can reply
+to existing topics by email. 
 
-If you want to suggest new functionality to be implemented in DHIS2 you
-can air your views on the developer mailing list and eventually write a
-specification, which is referred to as *blueprints* in Launchpad. The
-bueprint will be considered by the core development team and if accepted
-be assigned a developer, approver and release version. Blueprints can be
-browsed and added at *http://blueprints.launchpad.net/dhis2*
-
-If you find a bug in DHIS2 you can report it at Launchpad by navigating
-to *http://bugs.launchpad.net/dhis2* Your bug report will be
-investigated by the developer team and be given a status. If valid it
-will also be assigned to a developer and approver and eventually be
-fixed. Note that bugfixes are incorporated into the trunk and latest
-release branch - so more testing and feedback to the developer teams
-leads to higher quality of your software.
+The *Support* section of the Community of Practice includes all topics that were created using 
+DHIS2's previous collaboration platform, Launchpad, which is no longer active.
+
+Bugs identified on the Community of Practice should be submitted to the DHIS2 core team on *Jira*
 
 ## Reporting a problem
 
-If you encounter a problem with the DHIS2 software you can ask for
-assistance on the developer's mailing list at
-<https://launchpad.net/~dhis2-devs>. Before reporting, make sure that
-you have:
+If you encounter a problem while using DHIS2, take these steps first:
 
-  - Cleared the web browser cache (also called history or browsing data)
+  - Clear the web browser cache (also called history or browsing data)
     completely (select all options before clearing).
 
-  - Cleared the DHIS2 application cache: Go to Data administration -\>
+  - Clear the DHIS2 application cache: Go to Data administration -\>
     Cache statistics and click Clear cache.
```

Comment:
Data Administration -\> Maintenance -\> check "Clear application cache" and click "PERFORM MAINTENANCE"